grpclb.go

```diff
@@ -556,6 +618,13 @@ func (b *balancer) Get(ctx context.Context, opts BalancerGetOptions) (addr Addre
 				}
 				if !opts.BlockingWait {
 					b.next = next
+					if a.dropForLoadBalancing {
+						b.clientStats.NumCallsFinished++
```

Comment:
Optional: in the defer, if err != nil, do this `b.clientStats.NumCallsFinished++`.